src/Tasks/Task-2/GetRequest.js

```diff
@@ -5,21 +5,21 @@ import React, { useEffect, useState } from "react";
 const GetRequest = () => {
     const [info,setInfo] = useState([]);
     const [isLoading,setIsLoading] = useState(true);
+    const infofilter = [];
 
-    useEffect (()=>{
-         async function personInfo(){
-            await axios.get("https://randomuser.me/api/?results=12")
-            .then(response=>{
-                setInfo(response.data.results)
-            })
-           .then(info => {
-            setIsLoading(false);
-           })
-        }
+    const personInfo = async() =>{
+        const infoResult = await axios.get("https://randomuser.me/api/?results=12");
+        const inforResultArray = infoResult.data.results;
+        inforResultArray.map((el)=>{
```

Comment:
Use this syntax below to modify the state you will create above on line no. 8

const modifiedArray = await inforResultArray.map((el)=>{
return ({name:el.name,location:el.location,picture:el.picture})
setInfoFilter(modifiedArray)